misc/theme-verifier/src/test/resources/doubleSingleQuotes_en.properties

```diff
@@ -0,0 +1,17 @@
+#
```

Comment:
As per the mailing list, we no longer need to include copyright notices in every file.